# Remove the post install tasks because it's not properly installed when
# they run and portage does exactly the same things later anyway.

--- Rakefile.orig	2008-01-23 22:25:02.000000000 +0000
+++ Rakefile	2008-01-23 22:25:50.000000000 +0000
@@ -220,7 +220,5 @@
   system("gtk-update-icon-cache -f -t /usr/share/icons/hicolor") # HACK
 end
 
-task :post_install => [:scrollkeeper, :gconf, :update_icon_cache]
-
 
 #vim: filetype=ruby syntax=Ruby
